tensorboard/webapp/metrics/views/main_view/main_view_test.ts

```diff
@@ -363,6 +364,10 @@ describe('metrics main view', () => {
           sample: 0,
         },
       ]);
+      store.overrideSelector(
```

Comment:
My preference for adjusting this test would be to replace the override of getNonEmptyCardIdsWithMetadata and getCurrentRouteRunSelection with an override of getSortedRenderableCardIdsWithMetadata.